src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/part-product-list.js

```diff
@@ -0,0 +1,211 @@
+angular.module('virtoCommerce.DemoSolutionFeaturesModule')
+.controller('virtoCommerce.DemoSolutionFeaturesModule.partProductListController', [
+    '$scope',
+    'uiGridConstants',
+    'platformWebApp.uiGridHelper',
+    'platformWebApp.bladeUtils',
+    'virtoCommerce.DemoSolutionFeaturesModule.catalogProductApi',
+    function($scope, uiGridConstants, uiGridHelper, bladeUtils, productApi) {
+        $scope.uiGridConstants = uiGridConstants;
+        var blade = $scope.blade;
+        var filter = $scope.filter = blade.filter = {};
+        const bladeNavigationService = bladeUtils.bladeNavigationService;
+
+        blade.headIcon = 'fa-archive';
+
+        blade.currentEntity = angular.copy(blade.originalEntity);
+
+        blade.toolbarCommands = [
+            {
+                name: "platform.commands.add", icon: 'fa fa-plus',
+                executeMethod: () => $scope.addNewProduct(),
```

Comment:
If method doesn't return anything or you don't want to use the returned result, you should call it with braces.
() => { $scope.addNewProduct(); }

Reply:
done